decred/decred/dcr/account.py

```diff
@@ -1336,6 +1742,7 @@ def setPool(self, pool):
             self.addTxid(addr, txid)
         for utxo in bc.UTXOs([addr]):
             self.addUTXO(utxo)
+        self.updateSpentTickets()
```

Comment:
Does this need to be called every time `setPool` is called, or only when it's a new pool?

Reply:
I guess all of these only need to be called on a new pool.